examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -126,8 +131,18 @@ export interface ITaskListViewProps {
  */
 export const TaskListView: React.FC<ITaskListViewProps> = (props: ITaskListViewProps) => {
 	const { taskList } = props;
-
 	const [tasks, setTasks] = useState<ITask[]>(taskList.getTasks());
+	const setUnresolved = useSetRecoilState(unresolvedConflicts);
+
+	const trackChanges = (): void => {
```

Comment:
I believe this introduces another minor race condition. If this view is being launched for a new tab joining an existing collab session, there may initially be untracked changes, which we presumably want to display to the user right away (rather than waiting for the next task data state change).

To fix this, you could call this within the existing `useEffect` block - that way each time the effect runs (in this case, likely just the first time the component is run, since the `taskList` object doesn't ever change as far as I know), you'll appropriately update the `unresolved` state.